src/store/auth/auth-reducer.ts

```diff
@@ -5,13 +5,21 @@
 import { authActions, AuthAction } from "./auth-action";
 import { User } from "~/models/user";
 import { ServiceRepository } from "~/services/services";
+import { SshKeyResource } from '~/models/ssh-key';
 
 export interface AuthState {
     user?: User;
     apiToken?: string;
+    sshKeys?: SshKeyResource[];
```

Comment:
IMO, ssh keys should be a separate module. They are used to connect via CLI etc. Auth module is responsible for front end auth system.